unit_tests/test_bit_field_original_value.py

```diff
@@ -7,40 +7,22 @@
 
 @given("A BitField")
 def request_one_block(context):
-    context.uut = BitField(100, width=8)
+    context.uut = BitField(default_value=100, width=8, name="one_block")
 
 
 @given("A 4 byte BitField with value 100")
 def bitfield_ascii_100(context):
-    context.uut = BitField(100, width=32)
+    context.uut = BitField(default_value=100, width=32, name="ascii_100")
 
 
 @given(parsers.parse("A 4 byte BitField with value {value:d} and format ascii"))
 def bitfield_4_bytes(context, value):
-    context.uut = BitField(value, width=32, output_format="ascii")
-
-
-@given("Mutated once")
-def mutate_once(context):
-    context.uut.mutate()
-
-
-@given("Mutated twice")
-def mutate_twice(context):
-    context.uut.mutate()
-    context.uut.mutate()
-
-
-@given("Mutated thrice")
-def mutate_thrice(context):
-    context.uut.mutate()
-    context.uut.mutate()
-    context.uut.mutate()
+    context.uut = BitField(default_value=value, width=32, output_format="ascii", name="4_bytes")
 
 
 @when("Calling original_value")
 def call_original_value(context):
-    context.result = context.uut.original_value
+    context.result = context.uut.original_value().to_bytes(1, "little")
```

Comment:
`to_bytes()` is not available in Python 2, maybe we can use `struckt.pack()`?

Reply:
There's also the `str_to_bytes` function from helpers.py?